### ISSUE TITLE
Can't install webhook - CiviCRM 5.17.4

### ISSUE THREAD

**Issue by AidenGitHub** [cividesk/com.cividesk.email.sparkpost#88]
Hi there,

The webhook was not automatically installed, when I try to manually create it using https://mysite.com/civicrm?page=CiviCRM&q=civicrm/sparkpost/callback I get "POST to webhook target URL failed with error code EPROTO"

Any ideas?

Thanks!

**Comment by nganivet** [comment]
Q: did you replace "mysite.com" with the URL to your website? is the extensions URL correctly configured?

**Comment by AidenGitHub** [comment]
Yes the domain is correct and the extension resource URL is configured properly...
